package-lock.json

```diff
@@ -1,10 +1,6 @@
 {
   "name": "mx-design-tokens",
-<<<<<<< HEAD
-  "version": "8.1.1",
-=======
-  "version": "8.2.0",
->>>>>>> master
+  "version": "8.3.0",
```

Comment:
Delete this file, update the package.json as explained below and then run npm install.